source/layouts/resource.haml

```diff
@@ -3,29 +3,36 @@
 - @description = resource.meta_description || resource.snippet
 
 - content_for :header do
-  %header.aptible-header.aptible-header--with-breadcrumbs.hero-gradient--angled
-    = partial 'partials/main-nav', locals: {      |
-        section_title: 'Resources',               |
-        section_url: '/resources/'                |
-      }                                           |
+  %header.aptible-header.hero-gradient--angled
+    = partial 'partials/main-nav'
 
 = wrap_layout :layout do
-  .content.resource__header
-    .grid-container.grid-container--no-padding.resource__header-wrap{ style: resource_header_style(current_page) }
-      %h1.heading__title= resource.title
-      .heading__summary= resource.excerpt
-
   .content
-    .grid-container.grid-container--no-padding
-      .grid-document.markdown-content.resource-article
+    .grid-container
+      %header.blog-header
+        %h2.blog-header__title
+          %a{ href: '/resources/' } Resources
+        = partial 'follow-aptible-twitter-link'
+
+      .grid-document.resource-article
+        %header.blog-posts__post-header
+          %h1.resource-article__title= resource.title
+          .resource-article__subtitle= resource.excerpt
+
+        = partial 'partials/blog-social-share'
```

Comment:
Hackernews links in the contentful model work here just as they do on blog / changelog posts.